docs/DeveloperGuide.md

```diff
@@ -84,6 +85,8 @@ Follow [this guide](https://se-education.org/guides/tutorials/intellijCodeStyle.
 
 The **Architecture Diagram** given above explains the high-level design of the App.
 
+> 📝 **Note:** `DukeException` class is not shown in the diagram for simplicity-sake.
```

Comment:
Here "for simplicity-sake" should be written as "for simplicity's sake".

Reply:
Noted.